flink-runtime/src/main/scala/org/apache/flink/runtime/minicluster/LocalFlinkMiniCluster.scala

```diff
@@ -350,23 +350,18 @@ class LocalFlinkMiniCluster(
 
   def setMemory(config: Configuration): Unit = {
     // set this only if no memory was pre-configured
-    if (config.getInteger(ConfigConstants.TASK_MANAGER_MEMORY_SIZE_KEY, -1) == -1) {
+    if (config.getLong(TaskManagerOptions.MANAGED_MEMORY_SIZE) == -1L) {
```

Comment:
If we keep the default at -1 then we should compare against `TaskManagerOptons.MANAGED_MEMORY_SIZE.defaultValue()`; the same applies to other instances of this pattern.